src/components/Touchable.ios.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from "react";
+import { StyleSheet, TouchableOpacity } from "react-native";
+import type { TouchableProps } from "./TouchableTypes";
+import { TouchableDefaultProps } from "./TouchableTypes";
+
+const Touchable = ({ children, style, ...props }: TouchableProps) => (
+  <TouchableOpacity style={[styles.default, style]} {...props}>
+    {children}
+  </TouchableOpacity>
+);
+
+Touchable.defaultProps = {
+  ...TouchableDefaultProps,
+  delayPressIn: 50
```

Comment:
Can someone test this on a physical iPhone and check if `delayPressIn` is necessary? How does scrolling in list views feel without this?

Reply:
Had a look and this definitely makes a positive difference. With no delay, the press style is triggered almost every time when you try to start scrolling on a Touchable element.